apps/3000-home/pages/[...slug].js

```diff
@@ -0,0 +1,9 @@
+import { injectScript } from '@module-federation/utilities'
+import { createDynamicFederatedPage } from '@module-federation/nextjs-mf/utils'
+
+const { Page, getServerSideProps } = createDynamicFederatedPage({
+  injectScriptReplacement: injectScript
```

Comment:
Wasn't able to find how to fix an import issue for `injectScript` inside `nextjs-mf`, so temporarily I inject it from the outside

Reply:
Can you explain more or show me what it's supposed to look like. I'll pull branch and repro. I know my nx repo is a little difficult to link packages together.